contract-message-script/src/transaction_utils.sw

```diff
@@ -1,8 +1,9 @@
 library transaction_utils;
 
+use std::inputs::GTF_INPUT_MESSAGE_AMOUNT;
```

Comment:
Hmm. the spending script uses this file, and this file has changed. I feel like the script hash should have also changed as a result, but the hardcoded script hash in the predicate is still the same. What am I missing?

Reply:
The value for `GTF_INPUT_MESSAGE_AMOUNT` and `GTF_INPUT_CONTRACT_CONTRACT_ID` didn't change. We're just referencing them from the std lib now

Reply:
yeah makes sense.